faq.adoc

```diff
@@ -0,0 +1,76 @@
+<a href="url"><img src="https://d15n4q3o4x3svq.cloudfront.net/assets/tutorials/curl/api-a397cc184c5622fb5130af1b7baf149d.png" height="150" width="150" ></a>
+# Frequently Asked Questions
+This document contains answers to the most asked questions, if you have questions that are not answered
+in this document please refer to using our api-team email address: api@izettle.com .
+If you want to apply to receive access to our APIs, please fill in the form [here](https://docs.google.com/forms/d/e/1FAIpQLScdU9DDPfd680vhyZYAZ6DMIydQ1ikAOwP4abF2NWra5A-j8w/viewform)
+and we will come back to you with your credentials.
+
+---
+
+### General questions
+##### Question:
+> Does it cost anything to use your open APIs and is there some restriction on the API?
+
+##### Answer:
+
+> No, its completely free to use our API and the only limitation is that some resources and
+services might have rate-limiting enabled to prevent overload.
+
+___
+##### Question:
+> Do you provide any test environment accounts?
+
+##### Answer:
+> Unfortunately we do not at this time, there are however test accounts in production that you can use.
```

Comment:
No, the accounts listed in the authz documentation is for the sandbox environment we used initially when building authorization. That environment still work for testing the OAuth flow and some parts of purchase/finance but hasn't been updated in a long time and should probably be decommissioned.

There are no test accounts in production available to external integrators. They will need real accounts for the time being.